.github/workflows/full-test.yml

```diff
@@ -126,7 +126,7 @@ jobs:
         run: ./.github/run_tests.sh core
       - name: Test extractors
         if: ${{ steps.modules-changed.outputs.EXTRACTORS_CHANGED == 'true' || steps.modules-changed.outputs.CORE_CHANGED == 'true' }}
-        run: ./.github/run_tests.sh extractors
+        run: ./.github/run_tests.sh "extractors and not streaming_extractors"
```

Comment:
@alejoe91 They are not exclusive. That's why this is changed here. 

On that logic, what about the IBI extractors which are also streaming?

Reply:
Correct, maybe we should move the IBL streamer tests in the s3 file?